libs/sdk-ui-filters/src/AttributeFilter/Components/Addons/AttributeDatasetInfo.tsx

```diff
@@ -0,0 +1,37 @@
+// (C) 2023 GoodData Corporation
+
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { IMetadataObject } from "@gooddata/sdk-model";
+
+/**
+ * @internal
+ */
+export interface IAttributeDatasetInfoProps {
+    title?: string;
+    defaultAttributeFilterTitle?: string;
+    attributeDataSet?: IMetadataObject;
```

Comment:
Can we use `IDataSetMetadataObject`, to make the typing more proper/strict?

Reply:
Changed.